Engine/Graphics/Direct3D/Render.cpp

```diff
@@ -4187,8 +4187,8 @@ unsigned short *Render::MakeScreenshot(signed int width, signed int height) {
     float interval_x = game_viewport_width / (double)width;
     float interval_y = game_viewport_height / (double)height;
 
-    uint16_t *pPixels = (uint16_t *)malloc(2 * height * width);
-    memset(pPixels, 0, 2 * height * width);
+    uint16_t *pPixels = (uint16_t *)malloc(sizeof(uint16_t) * height * width);
+    memset(pPixels, 0, sizeof(uint16_t) * height * width);
 
```

Comment:
sizeof it's a constant and calculated in compile time, sizeof(uint16_t) == 2 in any case. Semantically equivalent, but less readable.

Reply:
The point is to use the the sizeof with the correct type according to the variable type, rather than a static number. Hypothetically if the sizeof(uint16_t) would ever change this code would still be valid, whilst having 2 not.